.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+sudo: required
+dist: xenial
+services:
+  - xvfb
+notifications:
+  email: false
+
+cache: pip
+
+# remember to remove 3.8 related workarounds below once
+# 3.8 is released and wheels are available. Also change 3.8-dev to 3.8
+
+addons:
+  apt_packages:
+    - pandoc
+    - libhdf5-dev # remove once h5py is packaged for 3.8
+    - gcc # this can go once we have scipy wheels for 3.8
+    - gfortran # this can go once we have scipy wheels for 3.8
+    - python-dev # this can go once we have scipy wheels for 3.8
+    - libopenblas-dev # this can go once we have scipy wheels for 3.8
+    - liblapack-dev # this can go once we have scipy wheels for 3.8
+
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8-dev"
```

Comment:
Please change this to "3.8" as in qcodes

Reply:
Done